suspend project capability, tests, validaitons

maheshKRex opened the pull request:
https://github.com/EmpowerPlastic/empowerchain/issues/396

gjermundgaraba reviewed (changes requested):
The only thing we need to add a test for is that is possible to approve a suspended project. Add that test to the approve project tests.

maheshKRex commented:
> The only thing we need to add a test for is that is possible to approve a suspended project. Add that test to the approve project tests.

added tests, and also fixed existing ones to check correct project id in the test case.
